test-packages/test-services/openapi/swagger-yaml-service/openapi/.openapi-generator/FILES

```diff
@@ -8,5 +8,4 @@ configuration.ts
 git_push.sh
 index.ts
 model/index.ts
-model/simple-test-entity.ts
```

Comment:
If you forget about this update, what will happen?

Reply:
What do you mean by "forget about this update"? This file is generated, I am not touching it.

Reply:
@jjtang1985 I'll already merge now, but let me know if you have concerns here, that we should address. I will then either correct it or create a ticket for this.